docs/account-management/connecting-bitbucket-and-semaphore.md

```diff
@@ -45,27 +45,27 @@ To disconnect your Bitbucket account:
 
 ## Checking the Connection Between Bitbucket and Semaphore
 
-To check the status of the connection between your Semaphore profile and Bitbucket, please check your [account settings](https://me.semaphoreci.com/account/).  
+To check the status of the connection between your Semaphore profile and Bitbucket, please check your [Account settings](https://me.semaphoreci.com/account/).  
 
 Your Bitbucket profile connection on this page can be in one of these states: 
 
 - **Not Connected** - Your account is not connected, when you try to log in again, you will need to grant Semaphore access again. 
 - **Connected** - You can connect both private and public repositories through OAuth App. 
 
-You can also check if the Semaphore OAuth app is installed by checking the the list of [authorized Oauth apps](https://bitbucket.org/account/settings/app-authorizations/) on your Bitbucket account. 
+You can also check if the Semaphore OAuth app is installed by checking the list of [authorized Oauth apps](https://bitbucket.org/account/settings/app-authorizations/) on your Bitbucket account. 
 
-**If you can see Semaphore OAuth app in the list, but you still cannot access your Semaphore account please contact our support team.**
+**If you can see the Semaphore OAuth app in the list, but you still cannot access your Semaphore account please contact our support team.**
```

Comment:
OAuth app -> OAuth App 

The word App should start with the capital A (it's like that everywhere in GHs docs).